src/dev/ci_setup/setup.sh

```diff
@@ -3,7 +3,8 @@
 set -e
 
 dir="$(pwd)"
-cacheDir="${CACHE_DIR:-"/tmp/kibana"}"
+workspace="$(cd "$dir"/..; pwd)"
+cacheDir="${CACHE_DIR:-"$workspace/.kibana"}"
```

Comment:
@elastic/infra - can you confirm the new path (`/var/lib/jenkins/workspace/.kibana`) is ok to use

Reply:
This won't work as intended, as the cache would be cleared out before each job run since the Jenkins `$WORKSPACE` gets purged. If you want a more permanent location, you could throw it under `${HOME}/.kibana`, although you'll want to ensure the directory exists before writing to it.

Reply:
Maybe I should change the variable name to "workspaces", as I believe it's causing confusion.

$WORKSPACE for the kibana job is  `/var/lib/jenkins/workspace/elastic+kibana+master+multijob-intake` where this cache would be a directory above that at `/var/lib/jenkins/workspace/.kibana`

Reply:
I will go ahead and change to the home directory as mentioned and move forward with that.

Reply:
I _think_ that with the clarification, _/var/lib/jenkins/workspace/.kibana_ would work, since _/var/lib/jenkins/workspace/{JOB_WORKSPACE}_ will be deleted with each new build, but I'd defer and go with whatever @elasticdog suggests when it comes to jenkins 😅